remote_monitoring_demo/README.md

```diff
@@ -34,3 +34,8 @@ $ ./remote_monitoring_demo.sh -r -o [kruize image]
 # To terminate the demo
 $ ./remote_monitoring_demo.sh -t
 
+# To run / restart the demo with a specific UI image
+# RUN:
+$ ./remote_monitoring_demo.sh -u [kruize ui image]
+# RESTART:
+$ ./remote_monitoring_demo.sh -r -u [kruize ui image]
```

Comment:
This shouldn't be here, it should be part of usage above and ui image should be optional

Reply:
made changes as suggested.  Thanks for the review @dinogun